### ISSUE TITLE
Cannot find the static files from when running the demo app. FileNotFoundError bootstrap.min.css

### ISSUE THREAD

**Issue by thesunlover** [GibbsConsulting/django-plotly-dash#137]
I have installed the dependedncies, and started the redis server. Then followed the configuration instructions in the `README.md` document and when I open the 7th demo I encounter an error
FileNotFoundError
--
[WinError 2] The system cannot find the file specified: 'D:\\dpd-env\\lib\\site-packages\\dpd_static_support\\bootstrap.min.css'

`dpd-env` is my virtualenv directory
![image](https://user-images.githubusercontent.com/6734600/56124293-fdcb7c00-5f7e-11e9-8d86-01f2b8e607df.png)

`pip freeze` data:
```
aioredis==1.2.0
alabaster==0.7.12
argh==0.26.2
asgiref==2.3.2
astroid==2.2.5
async-timeout==3.0.1
atomicwrites==1.3.0
attrs==19.1.0
autobahn==19.3.3
Automat==0.7.0
Babel==2.6.0
bleach==3.1.0
certifi==2019.3.9
channels==2.1.7
channels-redis==2.3.3
chardet==3.0.4
Click==7.0
colorama==0.4.1
constantly==15.1.0
coverage==4.5.3
daphne==2.3.0
dash==0.38.0
dash-bootstrap-components==0.4.0
dash-core-components==0.43.1
dash-html-components==0.13.5
dash-renderer==0.19.0
dash-table==3.5.0
decorator==4.4.0
Django==2.1.8
django-bootstrap4==0.0.8
-e git+https://github.com/thesunlover/django-plotly-dash@82191f85b00d78c90852338a4e21a14c8b5a034e#egg=django_plotly_dash
django-redis==4.10.0
docopt==0.6.2
docutils==0.14
dpd-components==0.1.0
dpd-static-support==0.0.2
Flask==1.0.2
Flask-Compress==1.4.0
grip==4.5.2
hiredis==1.0.0
hyperlink==19.0.0
idna==2.8
imagesize==1.1.0
incremental==17.5.0
ipython-genutils==0.2.0
isort==4.3.17
itsdangerous==1.1.0
Jinja2==2.10.1
jsonschema==3.0.1
jupyter-core==4.4.0
lazy-object-proxy==1.3.1
livereload==2.6.0
Markdown==3.1
MarkupSafe==1.1.1
mccabe==0.6.1
more-itertools==7.0.0
msgpack==0.6.1
nbformat==4.4.0
numpy==1.16.2
packaging==19.0
pandas==0.24.2
path-and-address==2.0.1
pathtools==0.1.2
pkginfo==1.5.0.1
plotly==3.7.1
pluggy==0.9.0
port-for==0.3.1
py==1.8.0
Pygments==2.3.1
PyHamcrest==1.9.0
pylint==2.3.1
pyparsing==2.4.0
pyrsistent==0.14.11
pytest==4.4.0
pytest-cov==2.6.1
pytest-django==3.4.8
python-coveralls==2.9.1
python-dateutil==2.8.0
pytz==2019.1
pywin32==224
PyYAML==5.1
readme-renderer==24.0
redis==3.2.1
requests==2.21.0
requests-toolbelt==0.9.1
retrying==1.3.3
six==1.12.0
snowballstemmer==1.2.1
Sphinx==2.0.1
sphinx-autobuild==0.7.1
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
tornado==6.0.2
tqdm==4.31.1
traitlets==4.3.2
twine==1.13.0
Twisted==19.2.0
txaio==18.8.1
typed-ast==1.3.1
urllib3==1.24.1
watchdog==0.9.0
webencodings==0.5.1
Werkzeug==0.15.2
whitenoise==4.1.2
wrapt==1.11.1
zope.interface==4.6.0
```

**Comment by GibbsConsulting** [comment]
Do you have ``dpd-static-support`` installed? Use ``pip freeze`` in the virtualenv to find out what is installed along with version info. Secondly, have you run (a Windows equivalent of) the ``prepare_demo`` script? Your error is consistent with missing one or both of these steps.

**Comment by thesunlover** [comment]
@GibbsConsulting thanks for the reply. I have just added the `pip freeze` results in my question above.
 I have run the Windows equivalents of commands inside the scripts mentioned in the instructions. there might be problems with versions of some of the used libs. If you have the option for using pyup bot it will keep updating the requirements each time a new version is released.

**Comment by GibbsConsulting** [comment]
The files `requirements.txt` (for django plotly dash itself) and `dev_requirements.txt` (for the demo site, along with build and test tools) are used when building the releases and also in the CI steps that run on every pull request, so they should be up to date.
Additionally, `frozen_dev.txt` is added to the codebase on each release - this is a historical record of the actual packages and their versions used.
Are you seeing some sort of version discrepancy?

**Comment by GibbsConsulting** [comment]
The issue is in the packaging of `dpd-static-support` - it is missing some of the css and js files

**Comment by GibbsConsulting** [comment]
@thesunlover there's a new version of `dpd-static-support` which should help address this - the older version on pypi was packaged with some files missing.
Can you try `pip install -U dpd-static-support` and then also rerun the `./manage.py collectstatic` step?

**Comment by GibbsConsulting** [comment]
#151 should have improved things also. Is this still an issue?

**Comment by Pooch11** [comment]
#136 Related?
I had this issue just recently, missing static files for bootstrap and renderer. After running manage.py collectstatic did not resolve my issue I peeked into my project folder and noticed a static folder outside of my app folder. Inside static there was admin, app and dash directories. I simply copied the dash directory (holding all the scripts) to the static directory under app.

Ill attach some screenshots to try and clarify

![staticfolder](https://user-images.githubusercontent.com/5437388/71266064-9fc51480-2315-11ea-9258-cc9c70291694.png)

![2019_12_20_10_36_58_static](https://user-images.githubusercontent.com/5437388/71265794-00078680-2315-11ea-96f7-9155e256a34d.jpg)

I don't know why it was put outside the app folder or if that was intended, but again, just moving dash to my static folder **under** the app directory got these embedded dashboards working. 

Perhaps it depends on the location that I run manage.py?

**Comment by GibbsConsulting** [comment]
Documentation updated in #255 and released as version 1.4.1
This should cover setting up the project. If not then please reopen this issue
